test_generator.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from pysize import get_size
+
+from generator import create_random_data_list, create_random_job_list
+
+
+def test_equal_size_data_list():
+    """Generate a lists of dicts and calculate right size."""
+    data_list = create_random_data_list()
+    data_job = create_random_job_list(data_list)
```

Comment:
It's actually two tests you've got here.

Reply:
@webknjaz Do you mean I need to divide into 2 parts this function?
Because I think now I just need to update name, documentation for that.

Reply:
yes